src/foam/net/node/RestDAOHandler.js

```diff
@@ -18,7 +18,7 @@
 foam.CLASS({
   package: 'foam.net.node',
   name: 'RestDAOHandler',
-  extends: 'foam.net.node.Handler',
+  extends: 'foam.net.node.BaseHandler',
 
```

Comment:
Docs would be handy, especially an overview describing its use relative to a RestDAO.

Reply:
Done.